gitlab/v4/objects.py

```diff
@@ -706,6 +706,16 @@ class GroupMemberManager(CRUDMixin, RESTManager):
     _update_attrs = (('access_level', ), ('expires_at', ))
 
 
+class GroupMemberAll(RESTObject):
+    pass
+
+
+class GroupMemberAllManager(ListMixin, RESTManager):
```

Comment:
I understand your goal here, but I think it would make more sense to add an `all()` method to the `GroupMemberManager` class. The `RunnerManager` class has the same logic. From a REST point of view `all` is an action applied to the resource at path `/groups/:id:/members` (GroupMember).

Does that make sense?

Reply:
Makes sense, hadn't seen that variation yet. Will try to change it like that soonish